response_parser.py

```diff
@@ -0,0 +1,47 @@
+def parse_weatherapi_response(weatherapi_response):
+    response_dict = weatherapi_response.json()
+    forecast_hours = response_dict['forecast']['forecastday'][0]['hour']
+    hours = []
+    temp = []
+    rain = []
+
+    # extracting needed information from request response
```

Comment:
Do you think we need this explanation here? I cannot see any of this kind in other methods.

Reply:
@KrzysiekJodlowski Done.